### PR TITLE
Arkansas inflation relief tax credit

### LINKED ISSUE #2971: Arkansas full-year resident tax credit
Inflationary Relief Income - Tax credit (Act 1 of the Third Extraordinary Session of 2022)
Allows for a full-year Arkansas resident to have a net income up to $101,000 an additional tax credit.
https://www.dfa.arkansas.gov/income-tax/individual-income-tax/whats-new/2022-whats-new/

tax credit worksheets:
https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_Final_InflationaryReliefIncomeTaxCredit_FI.pdf

### PR DESCRIPTION
**Pull request by Jingj-WANG**
Fixes #2971

### Summary
📄💵📊

This pull request adds three new parameter files to the policyengine-us repository to support the calculation of the inflationary relief income-tax credit in Arkansas. The files define the credit amount, the income limit, and the relevant metadata for each filing status based on the state's official sources.

> _`tax_credit_ark`_
> _New parameter files_
> _Inflationary relief_

### Walkthrough
*  Add new parameter files for the inflationary relief income-tax credit in Arkansas ([link](https://github.com/PolicyEngine/policyengine-us/pull/2974/files?diff=unified&w=0#diff-ed4341f2ba4042125b2336161fd2a7faf4e43be86a3b59deb07e5a9993cac84dL1-R75), [link](https://github.com/PolicyEngine/policyengine-us/pull/2974/files?diff=unified&w=0#diff-58cd0f0c33cd8fccc7ab8a362b9a36ebf59819103ee0f69658a657a836f57af6L1-R75), [link](https://github.com/PolicyEngine/policyengine-us/pull/2974/files?diff=unified&w=0#diff-4f0e33dfa27261c504b960e44851e392a5d1220c942f64623a4329cc13c8e92cL1-R20))
  - Define the credit amount for joint filers in `joint.yaml` based on the income-tax worksheet ([link](https://github.com/PolicyEngine/policyengine-us/pull/2974/files?diff=unified&w=0#diff-ed4341f2ba4042125b2336161fd2a7faf4e43be86a3b59deb07e5a9993cac84dL1-R75))
  - Define the credit amount for non-joint filers in `non_joint.yaml` based on the income-tax worksheet ([link](https://github.com/PolicyEngine/policyengine-us/pull/2974/files?diff=unified&w=0#diff-58cd0f0c33cd8fccc7ab8a362b9a36ebf59819103ee0f69658a657a836f57af6L1-R75))
  - Define the income limit for the credit in `income_limit.yaml` based on the income-tax worksheet ([link](https://github.com/PolicyEngine/policyengine-us/pull/2974/files?diff=unified&w=0#diff-4f0e33dfa27261c504b960e44851e392a5d1220c942f64623a4329cc13c8e92cL1-R20))
  - Provide description, metadata, reference, and brackets for each parameter file ([link](https://github.com/PolicyEngine/policyengine-us/pull/2974/files?diff=unified&w=0#diff-ed4341f2ba4042125b2336161fd2a7faf4e43be86a3b59deb07e5a9993cac84dL1-R75), [link](https://github.com/PolicyEngine/policyengine-us/pull/2974/files?diff=unified&w=0#diff-58cd0f0c33cd8fccc7ab8a362b9a36ebf59819103ee0f69658a657a836f57af6L1-R75), [link](https://github.com/PolicyEngine/policyengine-us/pull/2974/files?diff=unified&w=0#diff-4f0e33dfa27261c504b960e44851e392a5d1220c942f64623a4329cc13c8e92cL1-R20))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/amount.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/amount.yaml:1]: need a max_amount parameter with 150 and 300 (joint)

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/increment.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/increment.yaml:1]: make a reduction folder an put these three files in it

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/increment.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+description: Arkansas increases the income-tax credit for each of these increments of agi below the threshold, based on filing status.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/increment.yaml:1]:
```suggestion
description: Arkansas increases the income-tax credit for each of these increments of adjusted gross income below the threshold, based on filing status.
```

File: policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_incometax_credit.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        agi = tax_unit("adjusted_gross_income", period)
+        p = parameters(
+            period
+        ).gov.states.cr.tax.income.credits.inflationary_relief
+        filing_status = tax_unit("filing_status", period)
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_incometax_credit.py:17]: need the max amount that is being reduced

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions | page 26
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: INFLATIONARY RELIEF INCOME-TAX CREDIT WORKSHEET
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_Final_InflationaryReliefIncomeTaxCredit_FI.pdf
+  
+  breakdown:
+    - filing_status
+JOINT:
+  2022-01-01: 300
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml:15]: need to add 2021 values or a 0 is it was introduced in 2022

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+description: Arkansas has a maximum amount limit for income-tax credits, based on filing status.
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arkansas income-tax credit maximum amount 
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions | page 26
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml:7]:
```suggestion
    - title: Arkansas 2022 Individual Income Tax Forms and Instructions
```

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+description: Arkansas has a maximum amount limit for income-tax credits, based on filing status.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml:1]:
```suggestion
description: Arkansas allows for this maximum inflation relief credit amount, based on filing status.
```

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+description: Arkansas reduces the income-tax credit by this amount for each increment of adjusted gross income above the threshold, based on the filing status.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml:1]:
```suggestion
description: Arkansas reduces the inflation relief credit by this amount for each increment of adjusted gross income above the threshold, based on the filing status.
```

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arkansas income-tax credit reduce amount
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml:5]:
```suggestion
  label: Arkansas inflation relief credit reduction amount
```

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions | page 26
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: INFLATIONARY RELIEF INCOME-TAX CREDIT WORKSHEET
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_Final_InflationaryReliefIncomeTaxCredit_FI.pdf
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml:10]: all yaml files

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml [hunk 9]
```diff
@@ -0,0 +1,24 @@
+description: Arkansas reduces the income-tax credit by this amount for each increment of adjusted gross income above the threshold, based on the filing status.
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arkansas income-tax credit reduce amount
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions | page 26
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml:7]:
```suggestion
    - title: Arkansas 2022 Individual Income Tax Forms and Instructions 
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml:7]: all yaml- we only need the page in the href not in the title

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/increment.yaml [hunk 10]
```diff
@@ -0,0 +1,23 @@
+description: Arkansas reduces the income-tax credit for each of these increments of adjusted gross income above the threshold, based on filing status.
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arkansas income range increment
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/increment.yaml:5]:
```suggestion
  label: Arkansas income reduction increment
```

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/increment.yaml [hunk 11]
```diff
@@ -0,0 +1,23 @@
+description: Arkansas reduces the income-tax credit for each of these increments of adjusted gross income above the threshold, based on filing status.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/increment.yaml:1]:
```suggestion
description: Arkansas reduces the inflation relief credit for each of these increments of adjusted gross income above the threshold, based on filing status.
```

File: policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_incometax_credit.py [hunk 12]
```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class ar_income_tax_credits(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_incometax_credit.py:4]:
```suggestion
class ar_inflation_relief_credit(Variable):
```

File: policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_incometax_credit.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_incometax_credit.py:1]: rename file to ar_inflation_relief_credit.py

File: policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_incometax_credit.yaml
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_incometax_credit.yaml:1]: rename file to ar_inflation_relief_credit.yaml

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml [hunk 13]
```diff
@@ -0,0 +1,23 @@
+description: Arkansas provides income-tax credit to filers with adjusted gross income above this amount, based on filing status.
+  reference:
+    # There's no law files metioned about this new policy starting at Jan1 2022
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions | page 26
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: INFLATIONARY RELIEF INCOME-TAX CREDIT WORKSHEET
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_Final_InflationaryReliefIncomeTaxCredit_FI.pdf
+  label: Arkansas income-tax credit start
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml:8]:
```suggestion
  label: Arkansas inflation reduction credit reduction start
```

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml [hunk 14]
```diff
@@ -0,0 +1,23 @@
+description: Arkansas provides income-tax credit to filers with adjusted gross income above this amount, based on filing status.
+  reference:
+    # There's no law files metioned about this new policy starting at Jan1 2022
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml:3]:
```suggestion
    # The inflation relief credit is not mentioned in the Arkansas statutes 
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml:3]: add to all yaml files

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml [hunk 15]
```diff
@@ -0,0 +1,23 @@
+description: Arkansas provides income-tax credit to filers with adjusted gross income above this amount, based on filing status.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/start.yaml:1]:
```suggestion
description: Arkansas provides inflation relief credit to filers with adjusted gross income above this amount, based on filing status.
```

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml (hunk 3 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml:15]: instead we want a file called 

non_refundable.yaml - in credits 

with 2022-01-01: 
-ar_inflation...

File: policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_incometax_credit.py [hunk 16]
```diff
@@ -0,0 +1,26 @@
...
+        ).gov.states.cr.tax.income.credits.inflationary_relief
+        filing_status = tax_unit("filing_status", period)
+        max_amount = p.max_amount[filing_status]
+        reduction_start = p.reduction.start[filing_status]
+        increment = p.reduction.increment[filing_status]
+        reduction_per_increment = p.reduction.amount[filing_status]
+        excess = max_(agi - reduction_start, 0)
+        increments = np.ceil(excess / increment)
+        total_reduction_amount = increments * reduction_per_increment
+        return max_(max_amount - total_reduction_amount, 0)
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_incometax_credit.py:25]: add an adds function in the ar_non_refundable_credits file

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: need a git pull upstream master

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/non-refundable.yaml [hunk 17]
```diff
@@ -0,0 +1,11 @@
+description: Arkansas provides these nonrefundable income tax credits.
+values:
+  2022-01-01:
+    - ar_inflation_relief_credit
+metadata:
+  reference:
+    - title: Arkansas Income Tax Return Form 2022 AR1000F
+      href: "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_FullYearResidentIndividualIncomeTaxReturn.pdf"
+  unit: currency-USD
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/non-refundable.yaml:9]:
```suggestion
  unit: list
```

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/non-refundable.yaml [hunk 18]
```diff
@@ -0,0 +1,11 @@
+description: Arkansas provides these nonrefundable income tax credits.
+values:
+  2022-01-01:
+    - ar_inflation_relief_credit
+metadata:
+  reference:
+    - title: Arkansas Income Tax Return Form 2022 AR1000F
+      href: "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_FullYearResidentIndividualIncomeTaxReturn.pdf"
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/non-refundable.yaml:8]: add reference for all credits and if there is non just the reference for the inflation relief credit + page

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml [hunk 19]
```diff
@@ -0,0 +1,25 @@
...
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arkansas income-tax credit maximum amount 
+  reference:
+    # The inflation relief credit is not mentioned in the Arkansas statutes 
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions 
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: INFLATIONARY RELIEF INCOME-TAX CREDIT WORKSHEET | page1
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_Final_InflationaryReliefIncomeTaxCredit_FI.pdf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml:11]:
```suggestion
      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_Final_InflationaryReliefIncomeTaxCredit_FI.pdf#page=1
```

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml [hunk 20]
```diff
@@ -0,0 +1,25 @@
+description: Arkansas allows for this maximum inflation relief credit amount, based on filing status.
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arkansas income-tax credit maximum amount 
+  reference:
+    # The inflation relief credit is not mentioned in the Arkansas statutes 
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions 
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: INFLATIONARY RELIEF INCOME-TAX CREDIT WORKSHEET | page1
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml:10]:
```suggestion
    - title: INFLATIONARY RELIEF INCOME-TAX CREDIT WORKSHEET 
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml:10]: all yaml files

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml (hunk 19 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml:11]: all yaml files

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml [hunk 21]
```diff
@@ -0,0 +1,25 @@
...
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arkansas inflation relief credit reduction amount
+  reference:
+    # The inflation relief credit is not mentioned in the Arkansas statutes 
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions 
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: INFLATIONARY RELIEF INCOME-TAX CREDIT WORKSHEET | page1
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_Final_InflationaryReliefIncomeTaxCredit_FI.pdf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml:11]: add house bill: 
https://www.arkleg.state.ar.us/Home/FTPDocument?path=%2FAssembly%2F2021%2F2022S3%2FFiscal+Impacts%2FHB1002-DFA1.pdf#page=2

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml:11]: in reduction amount and increment add comment: 
# House bill did not specify reduction amount / increment

File: policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.yaml [hunk 22]
```diff
@@ -0,0 +1,44 @@
+- name: Single household with $101,002 agi
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.yaml:1]:
```suggestion
- name: Single household with $101,002 AGI
```

File: policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.yaml [hunk 23]
```diff
@@ -0,0 +1,44 @@
...
+- name: Joint household with $175,000 agi
+  period: 2022
+  input:
+    state_code: AR
+    adjusted_gross_income: 175_000
+    filing_status: JOINT
+  output:
+    ar_inflation_relief_credit: 280
+
+- name: Widow household with $80,000 agi
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.yaml:37]:
```suggestion
- name: Widow household with $80,000 AGI
```

File: policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.yaml [hunk 24]
```diff
@@ -0,0 +1,44 @@
...
+- name: Head of household with $98,200 agi
+  period: 2022
+  input:
+    state_code: AR
+    adjusted_gross_income: 98_200
+    filing_status: HEAD_OF_HOUSEHOLD
+  output:
+    ar_inflation_relief_credit: 30
+
+- name: Joint household with $175,000 agi
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.yaml:28]:
```suggestion
- name: Joint household with $175,000 AGI
```

File: policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.yaml [hunk 25]
```diff
@@ -0,0 +1,44 @@
...
+- name: Separate household with $100,000 agi
+  period: 2022
+  input:
+    state_code: AR
+    adjusted_gross_income: 100_000
+    filing_status: SEPARATE
+  output:
+    ar_inflation_relief_credit: 20
+
+- name: Head of household with $98,200 agi
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.yaml:19]:
```suggestion
- name: Head of household with $98,200 AGI
```

File: policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.yaml [hunk 26]
```diff
@@ -0,0 +1,44 @@
+- name: Single household with $101,002 AGI
+  period: 2022
+  input:
+    state_code: AR
+    adjusted_gross_income: 101_002
+    filing_status: SINGLE
+  output:
+    ar_inflation_relief_credit: 0
+
+- name: Separate household with $100,000 agi
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.yaml:10]:
```suggestion
- name: Separate household with $100,000 AGI
```

File: policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.py [hunk 27]
```diff
@@ -0,0 +1,25 @@
...
+class ar_inflation_relief_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arkansas inflation relief income-tax credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        agi = tax_unit("adjusted_gross_income", period)
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.py:13]:
```suggestion
        income = tax_unit("ar_taxable_income", period)
```

File: policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.py [hunk 28]
```diff
@@ -0,0 +1,25 @@
...
+class ar_inflation_relief_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arkansas inflation relief income-tax credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        income = tax_unit("ar_taxable_income", period)
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.py:13]: Can you add a note about this being the relevant income concept? The parameter description mentions AGI. It says line 28 so just connect that

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml [hunk 29]
```diff
@@ -0,0 +1,27 @@
+description: Arkansas allows for this maximum inflation relief credit amount, based on filing status.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml:1]:
```suggestion
description: Arkansas allows for this maximum inflationary relief tax credit amount, based on filing status.
```
This is the name in the worksheet so use it everywhere

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**